model/generate.go

```diff
@@ -387,8 +387,10 @@ func (g *GeneratedProject) addGeneratedProjectToConfig(intermediateProject *Pars
 						for j, intermediateProjectDT := range intermediateProjectBV.DisplayTasks {
 							if intermediateProjectDT.Name == dt.Name {
 								foundExisting = true
+								// avoid adding duplicates
+								_, execTasksToAdd := utility.StringSliceSymmetricDifference(intermediateProjectDT.ExecutionTasks, dt.ExecutionTasks)
```

Comment:
I think you want UniqueStrings. StringSliceSymmetricDifference will remove any execution tasks that are in both

Reply:
that is what i wanted, because I'm doing it before appending.